### DISCUSSION TITLE
certificate verification

### DISCUSSION THREAD

**Discussion by w-age** [open-meteo/open-meteo#673] [Q&A]
When using open-meteo API as a non-commercial user in a PyScript a warning is thrown: 
"Warning: Unverified HTTPS request is being made to host 'api.open-meteo.com'. Adding certificate verification is strongly advised."
Gemini suggests to download CA certificate from open-meteo and add it to the certificate store as the safest option to overcome this.
Assuming Gemini is right, where would I find such certifcate? Or, is this more of an issue for commercial users?

**Comment by patrick-zippenfenig** [comment]
Open-Meteo is using certificates from Let's encrypt. If you are using an older system or did not apply updates to the local certificate authority database, you could see this certificate error. Can you try to update your system and check again?

**Comment by w-age** [comment]
Thank you for your answer. I updated as per your advice and everything looks up-tp-date though still getting the warning. I reckon this is caused by the fact that my python code is executed in combination with pyscript.net. I don't fully overlook the underlying framework including WebAssembly, Emscripten and Pyodid and relevant setup/configuration of all involved components.
However, as I consider using a different framework I wont't investigate any further and - hopefully - will not encounter the issue once the alternative framework is set up. Thanks again!
